src/DotVVM.Framework/Compilation/Binding/ExpressionHelper.cs

```diff
@@ -85,6 +85,42 @@ static Expression GetDotvvmPropertyMember(Expression target, string name)
             );
         }
 
+        public static Expression SetMember(Expression node, Expression value)
```

Comment:
This method does more than just setting a normal member and for example should not be used for setting a member when assignment operator is used.

Reply:
Renamed it to `UpdateMember` and added a documentation comment.